src/ws/index.ts

```diff
@@ -0,0 +1,18 @@
+import { QueryClientConfig } from '../types';
+import { WebsocketClient, configureWebsocketClient } from './ws-client';
+
+/**
+ * Graasp websocket client top-level file
+ * Entry point to use the Graasp WebSocket client in front-end applications
+ */
+export * from './hooks';
+export { configureWebsocketClient } from './ws-client';
+export { default as MockWebSocket } from './mock-ws-client';
```

Comment:
Not sure if this should be an export in the production code. In an ideal tree shaking setup, the test code would never be part of the bundle, but listing it here will add it to the reachable tree.

Reply:
I agree with the issue of separating concerns but I think the goal here is to expose the MockWebsockets for use in app test code for example...